docs/pages/setup/helm-reference/teleport-plugin-jira.mdx

```diff
@@ -0,0 +1,314 @@
+
+You can [browse the source on GitHub](https://github.com/gravitational/teleport-plugins/tree/master/charts/access/jira).
+
+This reference details available values for the `teleport-plugin-jira` chart.
+
+(!docs/pages/includes/backup-warning.mdx!)
+
+## `teleport.address`
+
+| Type | Default value | Required? |
+| - | - | - |
+| `string` | `""` | Yes |
+
+This parameter contains the host/port combination of the teleport auth server.
+
+<Tabs>
+  <TabItem label="values.yaml">
+  ```yaml
+  teleport:
+    address: "teleport.example.com:3025"
+  ```
+  </TabItem>
+  <TabItem label="--set">
+  ```code
+  $ --set teleport.address="teleport.example.com:3025"
+  ```
+  </TabItem>
+</Tabs>
+
+## `teleport.identitySecretName`
+
+| Type | Default value | Required? |
+| - | - | - |
+| `string` | `""` | Yes |
+
+Name of the Kubernetes secret that contains the credentials for the connection.
+
+The secret should be in the following format:
+
+```yaml
+apiVersion: v1
+kind: Secret
+type: Opaque
+metadata:
+  name: teleport-plugin-jira-identity
+data:
+  auth_id: ...
+```
+
+Check out the [Jira Helm Chart documentation](https://github.com/gravitational/teleport-plugins/tree/master/charts/access/jira#prerequisites) for more information about how to acquire these credentials!
+
+<Tabs>
+  <TabItem label="values.yaml">
+  ```yaml
+  teleport:
+    identitySecretName: "teleport-plugin-jira-identity"
+  ```
+  </TabItem>
+  <TabItem label="--set">
+  ```code
+  $ --set teleport.identitySecretName="teleport-plugin-jira-identity"
+  ```
+  </TabItem>
+</Tabs>
+
+## `teleport.identitySecretPath`
+
+| Type | Default value | Required? |
+| - | - | - |
+| `string` | `"auth_id"` | No |
+
+Name of the key in the Kubernetes secret that holds the credentials for the connection. If the secret follows the format above, it can be omitted.
+
+<Tabs>
+  <TabItem label="values.yaml">
+  ```yaml
+  teleport:
+    identitySecretPath: "auth_id"
+  ```
+  </TabItem>
+  <TabItem label="--set">
+  ```code
+  $ --set teleport.identitySecretPath="auth_id"
+  ```
+  </TabItem>
+</Tabs>
+
+## `jira.url`
+
+| Type | Default value | Required? |
+| - | - | - |
+| `string` | `""` | Yes |
+
+Base URL of the Jira instance
+
+<Tabs>
+  <TabItem label="values.yaml">
+  ```yaml
+  jira:
+    url: "https://jira.example.com/"
+  ```
+  </TabItem>
+  <TabItem label="--set">
+  ```code
+  $ --set jira.url="https://jira.example.com/"
+  ```
+  </TabItem>
+</Tabs>
+
+## `jira.token`
+
+| Type | Default value | Required? |
+| - | - | - |
+| `string` | `""` | No |
+
+Jira token of the bot user to impersonate when sending the messages. Ignored when `jira.apiTokenFromSecret` is set.
+
+<Tabs>
+  <TabItem label="values.yaml">
+  ```yaml
+  jira:
+    token: "jiraapitoken"
+  ```
+  </TabItem>
+  <TabItem label="--set">
+  ```code
+  $ --set jira.token="jiraapitoken"
+  ```
+  </TabItem>
+</Tabs>
+
+## `jira.apiTokenFromSecret`
```

Comment:
I think it would help users navigate this guide more easily if we were to alphabetize the field names